eng/pipelines/cadl-ci.yml

```diff
@@ -22,33 +57,76 @@ stages:
         steps:
         - task: NodeTool@0
           inputs:
-            versionSpec: 16.14.2
+            versionSpec: '$(NodeVersion)'
           displayName: Install Node.js
         - task: Npm@1
           displayName: 'Install Dependencies for CADL csharp'
           inputs:
             command: install
-            workingDir: $(Build.SourcesDirectory)/autorest.csharp/src/CADL.Extension/Emitter.Csharp
+            workingDir: $(Build.SourcesDirectory)/src/CADL.Extension/Emitter.Csharp
         - task: Npm@1
           displayName: 'Build CADL csharp emitter'
           inputs:
             command: custom
             customCommand: run build
-            workingDir: $(Build.SourcesDirectory)/autorest.csharp/src/CADL.Extension/Emitter.Csharp
+            workingDir: $(Build.SourcesDirectory)/src/CADL.Extension/Emitter.Csharp
         - task: Npm@1
           displayName: 'Pack CADL Csharp emitter'
           inputs:
-            command: pack
-            workingDir: $(Build.SourcesDirectory)/autorest.csharp/src/CADL.Extension/Emitter.Csharp
-      - job: 'Unit_Test'
-        timeoutInMinutes: 120
-        steps:
-        - bash: |
-            npm install
-            npm run build
-            npm pack
-          displayName: 'Install package'
-          workingDirectory: $(Build.SourcesDirectory)/autorest.csharp/src/CADL.Extension/Emitter.Csharp
-        - pwsh: .\eng\CadlGenerate.ps1
+            command: custom
+            customCommand: pack
+            workingDir: $(Build.SourcesDirectory)/src/CADL.Extension/Emitter.Csharp
+        - pwsh: ./eng/CadlEmitterUT.ps1
```

Comment:
Will this step use the Emitter.csharp that is built in the previous step?

Reply:
yes